jenkins-pipelines/manager/ubuntu22-manager-backup.jenkinsfile

```diff
@@ -5,7 +5,7 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 managerPipeline(
     backend: 'aws',
-    region: 'us-west-2',
```

Comment:
this is for testing purposes ? or by mistake ?

Reply:
It's by purpose, otherwise the test would fail because of region and backup bucket misconfiguration.
Before, the test was passing as `backup_bucket_region` parameter was set to `us-east-1` (means that manager agent had this region in configuration) and we used backups from the bucket at the same region.